src/qml/SoundPage.qml

```diff
@@ -48,38 +47,23 @@ Item {
             id: muteButton
 
             anchors.fill: parent
-            onClicked: {
-                // Is muted?
-                if (preMuteLevel.value > 0) {
-                    // Restore pre mute volume value
-                    volumeControl.volume = preMuteLevel.value
-                    preMuteLevel.value = 0
-                } else {
-                    // Store volume value before muting
-                    preMuteLevel.value = volumeControl.volume
-                    volumeControl.volume = "0"
-                }
-            }
-
+            onClicked: [ volumeControl.volume, preMuteLevel.value ] = [ preMuteLevel.value, volumeControl.volume ]
         }
-    }
 
-    Icon {
-        id: volumeIcon
-        width: Dims.w(25)
-        height: width
-        anchors.fill: soundBackground
-        anchors.margins: Dims.l(3)
-        name: preMuteLevel.value > 0 ? "ios-sound-indicator-mute" :
-                              volumeControl.volume > "70" ? "ios-sound-indicator-high" :
-                                                            volumeControl.volume > "30" ? "ios-sound-indicator-mid" :
-                                                                                          volumeControl.volume > "0" ? "ios-sound-indicator-low" : "ios-sound-indicator-off"
+        Icon {
```

Comment:
Nesting the icon has an odd side effect where the icon also looks grayed out. This makes it look inconsistent with the Bluetooth page.

Reply:
You're right.  Fixed.